### ISSUE TITLE
Cents not being overwriiten

### ISSUE THREAD

**Issue by jrock2004** [pzuraq/ember-inputmask#44]
So I am using your plugin and when I type the numbers everything is getting replaced. Then I use the arrow keys to move before the cents symbol, hit the period and the cursor moves in front of first cent number. Then I try to hit a number and nothing happens. 

Here is a link to an Ember Twiddle with an example https://ember-twiddle.com/d7e72b96e9e48b66b4c16988d15a947e?openFiles=routes.application.js%2C

**Comment by brandynbennett** [comment]
@jrock2004 are you still seeing this? Your twiddle is set to 0.4.0-beta.5 and it seems to be working?

**Comment by jrock2004** [comment]
@brandynbennett I just tried it again with that link and it still does not work.

**Comment by brandynbennett** [comment]
You're right. It still does that. I think it may have something to do with the underlying library. A notable change from 0.4.0-beta.1 to beta.2 was a bump in the `inputmask` library. I made a completely new component `one-way-number-mask` that has a different architecture and the same bug is happening. Looking into it more
